Annoying getEpochBeaconCommittees test

There's one test that randomly fails 25% of the times it's run in CI. The maintainer of the test code could take a look and fix it if possible?

```
@chainsafe/lodestar:   1 failing
@chainsafe/lodestar:   1) beacon state api utils
@chainsafe/lodestar:        getEpochBeaconCommittees
@chainsafe/lodestar:          old/new epoch with epoch context:
@chainsafe/lodestar:      AssertionError: expected undefined not to be undefined
@chainsafe/lodestar:       at Context.<anonymous> (test/unit/api/impl/beacon/state/utils.test.ts:148:44)
@chainsafe/lodestar:       at processImmediate (internal/timers.js:439:21)
@chainsafe/lodestar:       at process.topLevelDomainCallback (domain.js:130:23)
```
From CI run https://github.com/ChainSafe/lodestar/runs/1506945865?check_suite_focus=true#step:9:1137

--- a/packages/lodestar/test/unit/api/impl/beacon/state/utils.test.ts
+++ b/packages/lodestar/test/unit/api/impl/beacon/state/utils.test.ts
@@ -142,7 +142,7 @@ describe("beacon state api utils", function () {
       const stateContext = getApiContext();
       stateContext.state = generateState({
         slot: 0,
-        validators: Array.from({length: 20}, () => generateValidator({activationEpoch: 1})) as List<Validator>,
+        validators: Array.from({length: 24}, () => generateValidator({activationEpoch: 1})) as List<Validator>,
       });
       const committees = getEpochBeaconCommittees(config, chainStub, stateContext, 1);
       expect(committees[0][0][0]).to.not.be.undefined;

